afs-core/src/main/java/com/powsybl/afs/AbstractNodeBase.java

```diff
@@ -177,12 +177,12 @@ private boolean nodeNameAlreadyExists(String name) {
         return childNodes.stream().filter(nodeInfo -> !nodeInfo.getId().equals(getId())).anyMatch(nodeInfo -> nodeInfo.getName().equals(name));
     }
 
-    public void archive(Path dir, boolean useZip, boolean archiveDependencies, Map<String, List<String>> outputBlackList) {
+    public void archive(Path dir, boolean useZip, boolean archiveDependencies, Map<String, List<String>> outputBlackList, Map<String, Boolean> keepTs) {
```

Comment:
use simply a list of pseudoClass that need its ts store not to be archive.
also keep the current method with no extra args and provide with default empty list.